contracts/aevm.md

```diff
@@ -23,19 +23,22 @@ Interaction with the first class objects of the Æternity chain (e.g.
 oracles, names, and state channels) is done through calls to a
 built-in contract at address 0.
 
+The value in the call indicates the tokens that the primop can use from the contract account.
+Unused tokens stay on the contract account.
```

Comment:
The above two lines are equivalent to stating that "the primop would check that tokens used by the primop is =< value".